ui/app/styles/components/access-control.scss

```diff
@@ -85,6 +85,26 @@
     max-height: 600px;
     overflow: auto;
   }
+
+  .editor-wrapper {
+    padding: 1rem 0;
+    &.error {
+      .CodeMirror {
+        box-shadow: 0 0 0 3px $red;
+      }
+      .help {
+        padding: 1rem 0;
+        font-size: 1rem;
+      }
+    }
+  }
+}
```

Comment:
Suggest renaming to namespace-editor-wrapper to disambiguate from (my fault, also vaguely named) variables editor wrapper on greps.

Reply:
sgtm!